src/segments/helm.go

```diff
@@ -0,0 +1,59 @@
+package segments
+
+import (
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type Helm struct {
+	props properties.Properties
+	env   platform.Environment
+
+	Version string
+}
+
+const (
+	// NewProp enables something
+	NewProp properties.Property = "newprop"
+)
+
+func (h *Helm) Enabled() bool {
+	displayMode := h.props.GetString(DisplayMode, DisplayModeAlways)
+	if displayMode != DisplayModeFiles {
+		return h.getVersion()
+	}
+
+	files := []string{"Chart.yml", "Chart.yaml"}
+	for _, file := range files {
+		_, err := h.env.HasParentFilePath(file)
+		if err != nil {
+			return false
```

Comment:
I think this will fail if we have a `Chart.yaml` and no `Chart.yml`. It will return early. That's why in my proposal, the boolean is outside of the loop and you set it to true and break when a match is found other than check on error.

Reply:
I missed that. I fixed that and also changed the test to cover that case. 
Thank you.